website/src/components/SearchPage/SeqPreviewModal.tsx

```diff
@@ -135,3 +133,29 @@ export const SeqPreviewModal: React.FC<SeqPreviewModalProps> = ({
         </Transition>
     );
 };
+
+interface DownloadButtonProps {
+    seqId: string;
+}
+
+const DownloadButton: React.FC<DownloadButtonProps> = ({ seqId }: { seqId: string }) => {
```

Comment:
As discussed below for the other page, I think this is confusing in that some of the items in this are buttons and now we have something that looks like those other buttons but isn't a button but a popover. But I don't really have a good suggestion for what to do about that in this case - and it's not too bad as it's not as "buttony" as the other page so probably let's leave it for now

Reply:
I had an idea on this - I think if we incorporate three vertical dots at the bottom right of the button that will help to indicate a menu will pop up

Reply:
I couldn't find a good icon in the icon library, I tried overlaying an ellipsis with CSS like ths: `<span className="custom-stroke absolute bottom-0.5 right-4 text-3xl">…</span>` but it didn't look nice, and I don't want to introduce a custom icon for this right now, so I'd say let's shelve it for now and get back to it again in the future?

Reply:
Thanks for trying! I have an idea that I would like to try but I'm hitting unrelated dev problems so I'll leave that for a future PR